packages/react-core/src/next/components/Wizard/WizardNavItem.tsx

```diff
@@ -96,11 +114,17 @@ export const WizardNavItem: React.FunctionComponent<WizardNavItemProps> = ({
             </span>
           </>
         ) : (
-          content
+          <>
+            {content}
+            {status === WizardNavItemStatus.Error && (
+              <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" className="pf-u-ml-sm" />
```

Comment:
Agreed. To be clear though, is this a blocking comment or something we can add back after this PR?

Reply:
I think it is fine to add as a follow up if @mcoker is ok with it.

Reply:
Fine with me, though it's worth noting that the utility class will not work unless the utility class CSS has been imported either in the component or into the user's app. Assuming we want it to work in the interim, it could be rewritten as...

```suggestion
            {status === WizardNavItemStatus.Error && (
              <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" style={{marginLeft: "var(--pf-global--spacer--sm)"}} />
```